### PR TITLE
ETFE-4234 Navigation added

### PR DESCRIPTION
**Pull request by grahampaulcook**
PR for app-config-base to house the new URL's added https://github.com/hmrc/app-config-base/pull/10673

PR for app-config-production to disable the message statistics notification https://github.com/hmrc/app-config-production/pull/22747

### CODE REVIEW

**Review by LewisMitchell — DISMISSED** [review]

**Comment by LewisMitchell** [comment]
Is it possible to have the layout.scala.html build a notification banner model rather than constructing it in each view?

**Comment by grahampaulcook** [comment]
> Is it possible to have the layout.scala.html build a notification banner model rather than constructing it in each view?

layout.scala.html doesn't have a `DataRequest` passed into it but instead a `Request` so would be unable to get hold of the ERN and message statistics to build that model. What has been done matches what was done for emcs-tfe-frontend so it's consistent with that frontend.

File: app/services/GetMessageStatisticsService.scala [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class GetMessageStatisticsService @Inject()(connector: GetMessageStatisticsConnector,
+                                            override val config: AppConfig
+                                           )(implicit ec: ExecutionContext) extends FeatureSwitching {
+
+  def getMessageStatistics(ern: String)(implicit hc: HeaderCarrier, request: UserRequest[_]): Future[Option[GetMessageStatisticsResponse]] = {
```
Comment (james-work-account) [inline app/services/GetMessageStatisticsService.scala:35]:
```suggestion
  def getMessageStatistics(ern: String)(implicit hc: HeaderCarrier): Future[Option[GetMessageStatisticsResponse]] = {
```
`request` is never used so can be removed

**Review by james-work-account — CHANGES_REQUESTED** [review]
Small change request. Otherwise LGTM

File: app/services/GetMessageStatisticsService.scala (hunk 1 above)
Comment (grahampaulcook) [reply app/services/GetMessageStatisticsService.scala]: Addressed at https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/350/commits/fa149054706fb8b20ca731b0d832b7e76fb43759

**Comment by james-work-account** [comment]
rebuild

**Review by james-work-account — APPROVED** [review]

**Pull request merged**